src/pages/Dashboard/scenes/Emails/Emails.tsx

```diff
@@ -1,9 +1,224 @@
-import React from 'react';
+import React, { useEffect, useState } from 'react';
+import EmailTemplate from '../../../../components/Dashboard/scenes/Emails/EmailTemplate';
+import EmailHistory from '../../../../components/Dashboard/scenes/Emails/EmailHistory';
+import { EMAILAPI_URL, EMAILAPI_SENDER } from '../../../../constants';
+import LoadingSmallSVG from '../../../../assets/svg/LoadingSmallSVG';
 
 const Emails: React.FC = () => {
+  const [selectedMentees, setSelectedMentees] = useState<string[]>([]);
+  const [subject, setSubject] = useState('');
+  const [body, setBody] = useState('');
+  const [view, setView] = useState('sent');
+  const [select, setSelect] = useState('');
+  const [isLoading, setIsLoading] = useState(false);
+  const [message, setMessage] = useState('');
+
+  const handleFormSubmit = async (e: React.FormEvent<HTMLFormElement>) => {
+    e.preventDefault();
+    const emailData = {
+      sender: `${EMAILAPI_SENDER}`,
+      recipients: selectedMentees,
+      subject,
+      body,
+    };
+    setIsLoading(true);
+    try {
+      const response = await fetch(`${EMAILAPI_URL}api/v1/send`, {
+        method: 'POST',
+        headers: {
+          'Content-Type': 'application/json',
+        },
+        body: JSON.stringify(emailData),
+      });
+
+      if (!response.ok) {
+        throw new Error(`HTTP error! status: ${response.statusText}`);
+      }
```

Comment:
Use a mutation here from react query

Reply:
ok i'll update this

Reply:
@anjula-sack, I'm working on this query mutation. Can you look at what I've implemented? It is not complete.😞